web/styles/dark_theme.css

```diff
@@ -401,6 +401,10 @@
         opacity: 0.5;
     }
 
+    .rendered_markdown .alert-word {
+        background-color: hsl(22deg 70% 35%);
```

Comment:
What I think we should do here is make an `--color-background-alert-word` CSS variable in `zulip.css`, one each for light and dark mode, and just reference that variable from `rendered_markdown.css`. Ideally `dark_theme.css` should only be tweaks that can't be handled by variables.

Reply:
Thanks for the review. I've updated the PR as per your instruction. Please check.